Probesys/bileto
# dev: Refactor the ticket activity

## Related issue(s)



https://github.com/Probesys/bileto/issues/34

## How to test manually



Scenario 1:

1. open a ticket with a user
2. assign the ticket to a technician → check the status changes to "in progress"
3. answer with the technician → check the status changes to "pending"
4. answer with the user → check the status changes to "in progress"
5. answer with the technician, adding a solution → check the status changes to "resolved" and that the message is marked as a solution
6. answer with the user → check the status changes back to "in progress" and that the message is no longer a solution
7. do the same by email

Scenario 2:

1. create an ongoing contract
2. open a ticket with a user → check the contract is associated to the ticket
3. do the same by email

Scenario 3:

1. open a ticket with some documents attached to the message → check that the attachments are attached to the message
2. do the same by email

## Checklist
